identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_ja_JP.properties

```diff
@@ -458,7 +458,13 @@ error.504=エラー 504 - ゲートウェイのタイムアウト
 
 ## Organization Login
 organization.login=組織ログイン
-organization.name=組織の名前
+organization.name=組織名
```

Comment:
Seems like there is an error when translating to Japanesee

Reply:
this is the correct translation given in google translate

Reply:
Ack